### PR TITLE
Add node 12 to CI builds

### PR DESCRIPTION
**Pull request by avh4**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 environment:
   ELM_VERSION: "0.19.0"
   matrix:
+  - nodejs_version: "12"
   - nodejs_version: "10"
   - nodejs_version: "8"
 
 platform:
-  # - x86
```
Comment (harrysarson) [inline appveyor.yml]: Out of interest: why do we not run tests on x86?

Comment (avh4) [reply appveyor.yml]: I'm assuming for the same reason I ran into in https://github.com/rtfeldman/node-elm-compiler/pull/98 -- that  elm@0.19.0-no-deps doesn't provide 32-bit Windows binaries, so I guess Evan doesn't want to support that platform any more?

Comment (harrysarson) [reply appveyor.yml]: Makes sense :+1:

**Comment by harrysarson** [comment]
@avh4 we can merge I think!

**Pull request merged**